test/js/setupTests.js

```diff
@@ -0,0 +1,74 @@
+const { spawn } = require("child_process");
```

Comment:
This looks great to me.  Just to make sure I'm understanding correctly, the reason you are going with jest here as opposed to rspec or minitest is for performance reasons? or is there something else that makes jest preferable?

Reply:
There are a couple of reasons. The #1 is speed - shelling out to `ruby -c` and `rubocop` and `minitest` takes a ton of time. I'd still like to bring that back, but only in an integration test capacity that doesn't need to run for the whole unit test suite. #2 is test coverage. It's really hard to measure test coverage when we're using ruby specs versus just using the stuff built into jest. #3 is the convenience of jest itself (`test.only`, `describe.each`, etc.). There's a lot of stuff baked in that makes it pretty nice to work with.

Reply:
Sounds good to me. Thanks for the explanation.